pkgs/games/liberal-crime-squad-git/default.nix

```diff
@@ -0,0 +1,28 @@
+{ fetchFromGitHub, stdenv, ncurses, autoconf, automake , SDL2, SDL2_mixer }:
+
+stdenv.mkDerivation rec {
+  version = "2016-03-03";
+  name = "liberal-crime-squad-git-${version}";
```

Comment:
I don't think `-git` suffix is needed (both there and in other places), unless there would also be non-git version of this in nixpkgs.